pages/1.10/networking/index.md

```diff
@@ -66,10 +65,10 @@ and tolerance to host failures.
 
 
 ## Edge-LB
-Edge-LB builds upon on HAProxy. HAProxy provides base functionality such as load balancing for TCP and HTTP-based applications, SSL support, and health checking. In addition, Edge-LB provides first class support for zero downtime service deployment strategies, such as blue/green deployment. Edge-LB subscribes to Mesos and updates HAProxy configuration in real time.
+[Edge-LB](/service-docs/edge-lb/0.1.9/) builds upon HAProxy. HAProxy provides base functionality such as load balancing for TCP and HTTP-based applications, SSL support, and health checking. In addition, Edge-LB provides first class support for zero downtime service deployment strategies, such as blue/green deployment. Edge-LB subscribes to Mesos and updates HAProxy configuration in real time.
 
 Edge-LB proxies and load balances traffic to all services that run on DC/OS. In contrast, Marathon-LB can only work with Marathon tasks. For example, if you are using Cassandra, Edge-LB can load balance the tasks launched by Cassandra.
```

Comment:
Edge-LB can be configured to proxy and load balance traffic to any service that benefits from having an upstream load balancer and running on DC/OS. In contrast, Marathon-LB can only work with Marathon tasks. For example, if you have a framework or a Marathon app, Edge-LB can load balance the their tasks launched by the framework or app.